src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitor.scala

```diff
@@ -18,13 +18,13 @@ import scala.util.{Success, Try, Failure}
  */
 object SubmissionMonitor {
   def props(workspaceName: WorkspaceName,
```

Comment:
passing in a submission which ultimately become outdated and needs to be reloaded anyway was strange